manifests/state-spectrum-x-operator/0040-spectrumx-operator.yaml

```
@@ -41,9 +41,11 @@ spec:
             - --health-probe-bind-address=:8081
             - --cm-namespace={{ .RuntimeSpec.Namespace }}
             - --cidrpools-namespace={{ .RuntimeSpec.Namespace }}
+            {{- if .CrSpec.SpectrumXConfig }}
```

Nit. Combine conditions
```suggestion
 {{- if and .CrSpec.SpectrumXConfig .CrSpec.SpectrumXConfig.Name }}
- --cm-name={{ .CrSpec.SpectrumXConfig.Name }}
{{- end }}
```

---

Done